<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="reset_button">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>330</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="show_time">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>300</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="moves_spent_label">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>270</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>MovesSpent : </string>
    </property>
   </widget>
   <widget class="QLabel" name="is_move_legal">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>240</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>IsMoveLegal (yes/no):</string>
    </property>
   </widget>
   <widget class="QPushButton" name="play_moves">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>370</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Autoplay</string>
    </property>
   </widget>
   <widget class="QLabel" name="points_label">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>210</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Points: </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pause">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>330</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QLabel" name="is_paused">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>180</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>isPaused: false</string>
    </property>
   </widget>
   <widget class="QLabel" name="is_won">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>150</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>haveYouWon: false</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
